packages/components/chip/src/variants/outlined.ts

```diff
@@ -117,11 +117,11 @@ export const outlinedVariants = [
   {
     design: 'outlined',
     hasClearButton: false,
-    class: tw(['px-md']),
+    class: tw(['px-[calc(var(--spacing-md)-var(--border-width-sm))]']),
```

Comment:
Is is possible to use `calc` ? IMO this is not good in terms of performance :( because more classes will be added to the end css bundle and the size would be computed in runtime when there is no need

Reply:
This class is aligned with the `dashed` design. Both designs shares the same classes so they are added nowadays.